.gitlab-ci.yml

```diff
@@ -1,54 +0,0 @@
-stages:
```

Comment:
I think it shouldn't be deleted

Reply:
I mean, we went for GitHub so leaving gitlab-ci file is imo kind-of legacy/unnecessary. However, you might be right, who knows if it will not be useful again, I'll add it back.

Reply:
Ah, you're right. It's github :D